app/(main)/league/[leagueId]/entry/[entryId]/week/Week.tsx

```diff
@@ -229,11 +263,32 @@ const Week = ({ entry, league, NFLTeams, week }: IWeekProps): JSX.Element => {
               {selectedLeague?.leagueName as string}
             </LinkCustom>
           </nav>
-          <section className="w-full pt-8" data-testid="weekly-picks">
+          <section className="flex flex-col items-center w-full pt-8" data-testid="weekly-picks">
             <h1 className="pb-8 text-center text-[2rem] font-bold text-foreground">
               Week {week} pick
             </h1>
 
+            {pickHistory.length > 0 && (
+              <section className="flex flex-wrap w-[90%] gap-4 overflow-x-scroll justify-center pb-10 items-center">
```

Comment:
Why does this have a w-[90%]?

Reply:
@ryan following along with what was below it for set width when showing all the available picks for that week.